Build/Chakra.Build.Default.props

```diff
@@ -3,10 +3,15 @@
   <Import Project="Common.Build.Default.props"/>
   <Import Condition="'$(Clang)'!=''" Project="Chakra.Build.Clang.Default.props"/>
   <PropertyGroup>
-    <WindowsTargetPlatformVersion Condition="'$(Platform)'=='ARM'">10.0.10240.0</WindowsTargetPlatformVersion>
+    <!-- Default the installed latest Win10 SDK -->
+    <WindowsTargetPlatformVersion_10>$(Registry:HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\Microsoft SDKs\Windows\v10.0@ProductVersion)</WindowsTargetPlatformVersion_10>
+    <!-- Sometimes the version in the registry has to .0 suffix, and sometimes it doesn't. Check and add it -->
+    <WindowsTargetPlatformVersion_10 Condition="'$(WindowsTargetPlatformVersion_10)' != '' and !$(WindowsTargetPlatformVersion_10.EndsWith('.0'))">$(WindowsTargetPlatformVersion_10).0</WindowsTargetPlatformVersion_10>
+    <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)'==''">$(WindowsTargetPlatformVersion_10)</WindowsTargetPlatformVersion>
 
-    <!-- Always use Platform SDK for core builds -->
-    <EventManifestXmlPath>$(WindowsSDK80Path)Include\um</EventManifestXmlPath>    
+    <!-- Unfortunately, ARM build depends on ksarm.h, which is only available in 10.0.10240.0 -->
```

Comment:
VS2017 installs "Windows Software Development Kit - Windows 10.0.15063.137" which is where the configuration is.
![apps](https://cloud.githubusercontent.com/assets/931192/26611170/38df9da6-4561-11e7-93d0-ca161c2988ab.PNG)